gcn/notices/chime/frb.detection.example.json

```
@@ -0,0 +1,24 @@
+{
+  "$schema": "https://gcn.nasa.gov/schema/main/gcn/notices/chime/frb.schema.json",
+  "alert_type": "initial",
+  "trigger_time": "2024-09-18T07:19:10.765268Z",
+  "trigger_time_error": [0.007864319719374001],
+  "id": ["427325191"],
+  "snr": 12.698559761047363,
+  "ra": 346.77850859547,
+  "dec": 12.632485229956252,
+  "ra_dec_error": [0.503806986334273, 0.598921231135782, 0],
+  "dm": 279.422607421875,
+  "dm_error": 0.40437427163124,
```

```suggestion
  "dm_error": [0.40437427163124],
```
This field needs to be an array